### PR TITLE
Fix check backend health

### PR DESCRIPTION
**Pull request by f1sh1918**
- add sudo for backend package postinstall
- fix tsc build: looks like due to merging issues some prop in a test was lost

### CODE REVIEW

**Comment by michael-markl** [comment]
Btw: You can trigger the `commit_main` pipeline through circleci, if you click on  the "Trigger pipeline" button (https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard?branch=fix-check-backend-health) and add the string parameter "run_commit_main" with  value "run".

File: .circleci/config.yml [hunk 1]
```diff
@@ -659,7 +659,7 @@ jobs:
                 path: ~/project
             - attach_workspace:
                 at: /tmp/workspace
-            - run: apt update -y && apt install -y nodejs npm curl
+            - run: apt update -y && apt install -y nodejs npm curl sudo
```
Comment (michael-markl) [inline .circleci/config.yml:662]: Possibly we should add `sudo` as a dependency to our debian package then?

Comment (michael-markl) [reply .circleci/config.yml:662]: https://github.com/digitalfabrik/entitlementcard/blob/release-mai-2024/.circleci/src/jobs/pack_backend.yml#L11C232-L11C255

**Comment by f1sh1918** [comment]
> Btw: You can trigger the `commit_main` pipeline through circleci, if you click on  the "Trigger pipeline" button (https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard?branch=fix-check-backend-health) and add the string parameter "run_commit_main" with  value "run".

OK i normally use command line for that

File: .circleci/config.yml (hunk 1 above)
Comment (f1sh1918) [reply .circleci/config.yml:662]: is that a feature circle-ci provides? i couldn't find any documentation how to do that in circleci @michael-markl 
Or are there specific debian image including sudo, which i also couldn't find in docker image documentation

Comment (ztefanie) [reply .circleci/config.yml:662]: Not really happy with this solution. Why do we want to add sudo, as everything runs on debian we can just remove the sudo's from the script. Also only adding sudo as a package seems not to be sufficent, as we need to add the current user to sudoers group and log in and out.

**Review by ztefanie — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (f1sh1918) [reply .circleci/config.yml:662]: currently when i try to switch user on staging server i need sudo. thats why the command is like that for migration in the post install script

Comment (michael-markl) [reply .circleci/config.yml:662]: > is that a feature circle-ci provides? i couldn't find any documentation how to do that in circleci @michael-markl 

The line I referred to uses the pack deb script which accepts dependencies for the deb package. Currently the only dependency is the Java runtime. We could add sudo there.

(Sorry for slow response, I'm on vacation)

**Pull request merged**